### DISCUSSION TITLE
Congrats @arpittyagi102 🥳

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#20] [Congrats]
Congratulations @arpittyagi102 for earning a `Galaxy Brain` by answering in [Galaxy Qs](https://github.com/Sriansh-raj/GitHub-Achievements/discussions/categories/galaxy-qs)

❤️

**Comment by arpittyagi102** [comment]
Thank you so much 😊😊😊😊
